sale_delivery_block/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import sale_order
+from . import sale_delivery_block_reason
```

Comment:
sale_delivery_block_reason file should be loaded earlier than sale_order

Reply:
why?

Reply:
There is a manye2one reference in sale_order file and probably thats why it should come after sale_delivery_block_reason. So the order of import stays justified. I just had an experience of a trouble in one another such case

Reply:
That would be odd. In my experience, Odoo is able to instantiate the models in a module regardless of loading order even if there are cross references in relational fields. And tests are green so that happens here as well.